### PR TITLE
Cleanup warnings and formatting

### PR DESCRIPTION
**Pull request by yishaigalatzer**
Remove warnings from package management

### CODE REVIEW

**Comment by pranavkm** [comment]
:shipit:

File: src/ProjectManagement/NuGetProject.cs [hunk 1]
```diff
@@ -15,7 +15,7 @@ public abstract class NuGetProject
         protected NuGetProject() : this(new Dictionary<string, object>()) { }
         protected NuGetProject(IDictionary<string, object> metadata)
         {
-            if(metadata == null)
+            if (metadata == null)
             {
                 throw new ArgumentNullException("metadata");
```
Comment (deepakaravindr) [inline src/ProjectManagement/NuGetProject.cs]: Did you intend to change this to nameof(metadata) as well? If not, we can always fix this using re-sharper

File: src/ProjectManagement/Projects/FolderNuGetProject.cs [hunk 2]
```diff
@@ -93,10 +93,10 @@ public override Task<IEnumerable<PackageReference>> GetInstalledPackagesAsync(Ca
             return true;
         }
 
-        public async override Task<bool> UninstallPackageAsync(PackageIdentity packageIdentity, INuGetProjectContext nuGetProjectContext, CancellationToken token)
+        public override Task<bool> UninstallPackageAsync(PackageIdentity packageIdentity, INuGetProjectContext nuGetProjectContext, CancellationToken token)
         {
             // Do nothing. Return true
-            return true;
+            return Task.FromResult(true);
```
Comment (deepakaravindr) [inline src/ProjectManagement/Projects/FolderNuGetProject.cs]: This one is me :)

**Comment by deepakaravindr** [comment]
:shipit:

File: src/ProjectManagement/NuGetProject.cs (hunk 1 above)
Comment (yishaigalatzer) [inline src/ProjectManagement/NuGetProject.cs]: There is a resharper fix Andriy sent out for review

**Pull request closed without merging**